packages/atomic-react/package.json

```diff
@@ -12,15 +12,14 @@
     "build": "nx build",
     "clean": "rimraf -rf dist",
     "build:bundles:esm": "tsc -p tsconfig.esm.json",
-    "build:bundles:cjs": "tsc -p tsconfig.cjs.json",
-    "build:bundles:iife": "rollup --config rollup.config.mjs",
-    "build:bundles": "concurrently \"npm run build:bundles:esm\" \"npm run build:bundles:cjs\"  \"npm run build:bundles:iife\"",
+    "build:bundles:iife-cjs": "rollup --config rollup.config.mjs",
```

Comment:
Rollup is now used to generate both `cjs` and `iife` bundles hence the name change.